### DISCUSSION TITLE
Use latest versions of Paste for Flex plugin development

### DISCUSSION THREAD

**Discussion by twilio-jyoung** [twilio-labs/paste#2752] [Q&A] [answered]
Has anyone figured out a way to get this to work?

I am trying to use the latest paste packages which have components that I would like to use.  However, Flex UI 2.0 references

`{"dependencies": {
		"@twilio-paste/core": "^10.20.0",
		"@twilio-paste/icons": "^5.7.0",
	}}`

Trying to update to the latest versions breaks peer dependencies and fails.

![Screen Shot 2022-10-19 at 2 56 06 PM](https://user-images.githubusercontent.com/44909531/196791290-aba0bdfd-eb64-452d-92b9-61dcabd8d795.png)

**Comment by gloriliale** [comment]
Hi @twilio-jyoung!

The folks at #help-flex-ui20 would be your best bet for figuring this out. Many thanks!

**Reply by moretalk** [reply]
@gloriliale Great for Twilio Staff. Less helpful for partners 👎🏻 Can we get access as a Twilio Gold Partner? 👍🏻

**Comment by twilio-jyoung** [comment, marked as answer]
`npm install @twilio-paste/core@16.0.1 @twilio-paste/icons@9.3.0 --legacy-peer-deps` seems to have done the trick

**Reply by gloriliale** [reply]
Thanks for posting this update @twilio-jyoung!

**Reply by moretalk** [reply]
@gloriliale No longer the answer for Version 18+ sadly :(
